api/api.py

```diff
@@ -24,31 +23,32 @@ def get_book_details():
     try:
         isbn = soup.find("span", itemprop="isbn").text
         title = soup.find(class_="book_title").text
```

Comment:
What will happen if the developer of the website change the class? How can we notice that this script does not work anymore?